qemu/tests/cfg/qemu_guest_agent.cfg

```
@@ -409,6 +409,14 @@
             gagent_check_type = user_logoff
         - gagent_qgastatus_after_remove_qga:
             gagent_check_type = qgastatus_after_remove_qga
+        - gagent_container:
+            only Linux
+            no RHEL.8.0, RHEL.8.1
+            gagent_check_type = container
+            gagent_container_start_cmd = "podman start qemu-guest-agent"
+            get_target_ver_cmd = "cat /refer_ver.txt"
+            get_container_ver_cmd = "podman inspect qemu-guest-agent | grep %s"
+            # Please define create_container_cmd before this test: "create_container_cmd = podman create --name qemu-guest-agent --network xxx --privileged --volume xxx:xxx --pull missing qemu-guest-agent:xxx"
```

Better break the lines since it seems too long.
```suggestion
            # Please define create_container_cmd before this test:
            # create_container_cmd = podman create --name qemu-guest-agent --network xxx --privileged --volume xxx:xxx --pull missing qemu-guest-agent:xxx
```

---

okay, got it.